R/pkg/inst/tests/testthat/test_mllib.R

```diff
@@ -141,3 +141,17 @@ test_that("kmeans", {
   cluster <- summary.model$cluster
   expect_equal(sort(collect(distinct(select(cluster, "prediction")))$prediction), c(0, 1))
 })
+
+test_that("survreg vs survival::survreg", {
+  data <- list(list(4,1,0,0), list(3,1,2,0), list(1,1,1,0),
+          list(1,0,1,0), list(2,1,1,1), list(2,1,0,1), list(3,0,0,1))
+  df <- createDataFrame(sqlContext, data, c("time", "status", "x", "sex"))
+  model <- survreg(Surv(time, status) ~ x + sex, df)
+  stats <- summary(model)
+  coefs <- as.vector(stats$coefficients[,1])
+  rCoefs <- c(1.3149571, -0.1903409, -0.2532618, -1.1599802)
```

Comment:
We should load library survival if using R's ```survreg``` to training a control model, and the test environment may not with survival package installed. So I just paste the R's output here and compared with ```SparkR::survreg```.